### ISSUE TITLE
use suffix in generated IRI

### ISSUE THREAD

**Issue by EnnoMeijers** [netwerk-digitaal-erfgoed/LDWizard-ErfgoedWizard#3]
Would is be possible to add support for adding a suffix to the IRI like: http://data.netwerkdigitaalerfgoed.nl/literaireprijzen/pers<rownr>?

**Comment by wouterbeek** [comment]
This would already work when the base IRI is set to `http://data.netwerkdigitaalerfgoed.nl/literaireprijzen/` and the header label is `pers`.  But you must mean something else here...

**Comment by EnnoMeijers** [comment]
Yes sorry for not being precise here. I was thinking of generating an IRI based on the row number (or column value) with the following appearance:
http://data.netwerkdigitaalerfgoed.nl/literaireprijzen/pers1
http://data.netwerkdigitaalerfgoed.nl/literaireprijzen/pers2
...
http://data.netwerkdigitaalerfgoed.nl/literaireprijzen/persN

This is done sometimes to mint IRIs based on specific types without adding extra layers in the IRI. Maybe we should discuss the need for support for this in a broader setting?

**Comment by wouterbeek** [comment]
We currently support the use of the row number at the end of generated IRIs, but after a forward slash.  See [this issue](https://github.com/netwerk-digitaal-erfgoed/LDWizard-ErfgoedWizard/issues/21) for the way in which we create subject IRIs.
